From bc911faf39ebda5bd5d929dcc60bf294ce7e1760 Mon Sep 17 00:00:00 2001
From: Russell King <rmk+kernel@arm.linux.org.uk>
Date: Fri, 15 May 2015 18:13:27 +0100
Subject: [PATCH 099/149] ARM: imx6: add power domains to Vivante GPU nodes

v4.1 requires the power domains to be specified for Vivante GPU nodes,
otherwise they fail to be detected.

Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 arch/arm/boot/dts/imx6qdl.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 07b98ce..018975b 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -127,6 +127,7 @@
 			clocks = <&clks IMX6QDL_CLK_GPU2D_AXI>,
 				 <&clks IMX6QDL_CLK_GPU2D_CORE>;
 			clock-names = "bus", "core";
+			power-domains = <&gpc 1>;
 		};
 
 		gpu_3d: gpu@00130000 {
@@ -137,6 +138,7 @@
 				 <&clks IMX6QDL_CLK_GPU3D_CORE>,
 				 <&clks IMX6QDL_CLK_GPU3D_SHADER>;
 			clock-names = "bus", "core", "shader";
+			power-domains = <&gpc 1>;
 		};
 
 		hdmi: hdmi@0120000 {
-- 
2.6.2

